test/src/main/configurations/FileHandling/Configuration_MFH.xml

```diff
@@ -12,7 +12,7 @@
 				minStableTime ="${mfh.flow1.minstabletime}"
 				fileTimeSensitive="true"
 				name="MFHIn"
-				messageType="info"
+				messageType="INFO"
```

Comment:
If using an enum means the value now has to be uppercase then that is another backwards incompatible change.

But doesn't the configuration parser automatically parse enums case-insensitive?

(Backwards incompatible changes shouldn't be in minor releases, right?)

Reply:
I tested a lowercase enum locally and it seems to work correctly, so that is backwards compatible. The only breaking change is the one mentioned above where I removed the posibility to get attributes or headers based on a search key.

Reply:
Thanks, I wanted to make sure of this.

The one change mentioned above that is breaking, I think that it something we can get away with in practice because I don't think that anyone actually uses any of the other possible values.